src/targets/matekf411rx_fs/target.h

```diff
@@ -0,0 +1,65 @@
+#include "config.h"
+
+#define MatekF411RX_FS
+
+// PORTS
+#define SPI_PORTS   \
+  SPI1_PA5PA6PA7    \
+  SPI2_PB13PB14PB15 \
+  SPI3_PB3PB4PB5
+
+#define USART_PORTS \
+  USART1_PA10PA9    \
+  USART2_PA3PA2
+
+// LEDS
+#define LED_NUMBER 1
+#define LED1PIN PIN_C13
+#define LED1_INVERT
+
+#define BUZZER_PIN PIN_C15
+#define BUZZER_INVERT
+
+//#define FPV_PIN LL_GPIO_PIN_13
+//#define FPV_PORT GPIOA
+
+// GYRO
+#define GYRO_SPI_PORT SPI_PORT1
+#define GYRO_NSS PIN_A4
+#define GYRO_INT PIN_A1
+#define GYRO_ORIENTATION GYRO_ROTATE_90_CCW
+
+// RADIO
+// PB3/PB4/PB5 (SPI3) is used for spi connection to A7105
+// PA15 is used for SPI CSN(NSS)
+// PA14 is used as external interrupt with rising edge trigger connected to A7105
+// PB9 is used as output (push/pull, no pullup/pulldown) for RX LED
+// PB2 is used as input (no pullup/pulldown) for RX bind pushbutton
+#define USE_A7105
+#define A7105_SPI_PORT SPI_PORT3
+#define A7105_NSS_PIN PIN_A15
+#define A7105_GIO1_PIN PIN_A14    // EXTI15_10_IRQHandler
+#define RX_LED_PIN PIN_B9
+#define RX_BIND_PIN PIN_B2
+
+#ifdef SERIAL_RX
```

Comment:
deprecated, lets drop this including the RX_USART assigment

Reply:
Removed RX_USART definition with 993df7e